convert_hf_to_gguf.py

```diff
@@ -453,8 +453,11 @@ def from_model_architecture(cls, arch: str, model_type = ModelType.TEXT) -> type
 
 
 class TextModel(ModelBase):
+    model_type = ModelType.TEXT
```

Comment:
```suggestion
    model_type = ModelType.TEXT
    hf_arch: str
```

Not sure if this is the reason why the CI is not happy

Reply:
Done.